### PR TITLE
Add a new API GetStats to get stats value for places given statistical variable

### PR DESCRIPTION
**Pull request by shifucun**

### CODE REVIEW

File: store/populations.go [hunk 1]
```diff
@@ -471,3 +473,147 @@ func iterateSortPVs(pvs []*pb.PropertyValue, action func(i int, p, v string)) {
...
+			continue
+		}
+		if obs.MeasuredProp != mprop {
+			continue
+		}
+		msg := proto.MessageReflect(obs)
+		fd := msg.Descriptor().Fields().ByJSONName(statType)
+		if msg.Has(fd) {
+			ts.Data[obs.ObservationDate] = msg.Get(fd).Float()
+		}
```
Comment (pradh) [inline store/populations.go:505]: Does this work? I thought the current ObsTimeSeries proto does not have these fields. Instead mmethod, mprop are part of the key.

Comment (shifucun) [reply store/populations.go:515]: As discussed, I added all the observation fields here to check. Also using the StatisticalVariable proto here.

File: store/populations.go [hunk 2]
```diff
@@ -471,3 +473,163 @@ func iterateSortPVs(pvs []*pb.PropertyValue, action func(i int, p, v string)) {
 		action(i, p, pvMap[p])
 	}
 }
+
+func addObsSeries(
+	key string,
+	cacheValue string,
+	result map[string]pb.ObsTimeSeries,
```
Comment (Spaceenter) [inline store/populations.go:480]: result map[string]*pb.ObsTimeSeries,
use pointer

File: store/populations.go [hunk 3]
```diff
@@ -471,3 +473,163 @@ func iterateSortPVs(pvs []*pb.PropertyValue, action func(i int, p, v string)) {
...
+	statsVar pb.StatisticalVariable,
```
Comment (Spaceenter) [inline store/populations.go:481]: statsVar *pb.StatisticalVariable

use pointer

File: store/populations.go [hunk 4]
```diff
@@ -471,3 +473,163 @@ func iterateSortPVs(pvs []*pb.PropertyValue, action func(i int, p, v string)) {
...
+) error {
+	parts := strings.Split(key, "^")
+	dcid := strings.TrimPrefix(parts[0], util.BtObsSeriesPrefix)
+	val, err := util.UnzipAndDecode(cacheValue)
+	if err != nil {
+		return err
+	}
+	pbData := pb.PopObsPlace{}
```
Comment (Spaceenter) [inline store/populations.go:489]: pbData := &pb.PopObsPlace{}
	jsonpb.UnmarshalString(string(val), pbData)

Comment (shifucun) [reply store/populations.go]: Done

File: store/populations.go (hunk 3 above)
Comment (shifucun) [reply store/populations.go]: Done

**Pull request merged**